modules/contact/server-ts/resolvers.ts

```diff
@@ -13,10 +15,12 @@ interface ContactInput {
 export default () => ({
   Mutation: {
     async contact(obj: any, { input }: ContactInput, { mailer, req: { t } }: any) {
-      const errors = new FieldError(validate(input, contactFormSchema));
+      // TODO if import ApolloError that get error 'app is not defined'
+      const { ApolloError, UserInputError } = require('apollo-server-express');
 
-      if (errors.hasAny()) {
-        return { errors: errors.getErrors() };
+      const errors = validate(input, contactFormSchema);
+      if (!isEmpty(errors)) {
+        throw new UserInputError('Failed to get events due to validation errors', { errors });
```

Comment:
All the texts coming from the server should be translated.

Reply:
Add translated for error